docs/components/integration-framework/connectors/out-of-the-box-connectors/available-connectors-overview.md

```diff
@@ -6,8 +6,9 @@ description: Take a closer look at the Connectors available in Camunda Platform
 
 This section gives an overview of the **out-of-the-box Connectors** available in Camunda Platform 8:
 
-- [REST Connector](rest.md) - Make a request to a REST API and use the response in the next steps of your process.
-- [SendGrid Connector](sendgrid.md) - Quickly send emails from your BPMN processes.
-- [Slack Connector](slack.md) - Send messages to channels or users in your [Slack](https://slack.com) workspace from your BPMN process.
+- [Google Drive connector](googledrive.md) - Create empty folders or documents on [Google Drive](https://www.google.com/drive/) from templates from your BPMN process.
```

Comment:
Update text to: 
"Create folders or files from a [Google Drive] (https://www.google.com/drive/) template from your BPMN process."

Reply:
Also can the order of the list be: 
1. REST Connector
2. SendGrid Connector
3. Slack Connector
4. Google Drive Connector
(For our other new connectors coming)
5. AWS Simple Queue Service Connector
6. AWS Lambda Connector

Reply:
Addressed in a new revision. Will publish shortly.

Regarding connectors order - there was a suggestion from Max to keep them alphabetical.

Reply:
Ok cool (on the alphabetical) piece!